### PR TITLE
fixes #280 - Added word clicking feature to loklakwordcloud app.

### LINKED ISSUE #280: Add word clicking feature to loklakwordcloud app
Add feature to click a word. The current word cloud will be replaced with clicked word's cloud.

### PR DESCRIPTION
**Pull request by djmgit**
fixes issue #280, on clicking a word, the cloud will be updated with the
selected word's cloud.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link : http://loklakapps.surge.sh/LoklakWordCloud/

@singhpratyush @kavithaenair @Achint08 @SKrPl @vibhcool please review :)

File: LoklakWordCloud/js/script.js [hunk 1]
```diff
@@ -67,7 +68,21 @@ app.controller("app", function ($scope, $http) {
             data = {};
             data["text"] = word;
             data["weight"] = $scope.wordFreq[word];
-            $scope.wordCloudData.push(data);
+            data["handlers"] = {
+                click: function() {
+                    alert("haha");
```
Comment (Achint08) [inline LoklakWordCloud/js/script.js]: why this `alert("haha")'`?

**Comment by djmgit** [comment]
@Achint08 made changes, please re-review

**Comment by vibhcool** [comment]
Numbers doesn't look good.
screenshot from 2017-08-01 05-00-38

**Comment by vibhcool** [comment]
Also words in `stopwords` can be a movie name. Could this stopwords be solved by setting a lower limit to those words after which they can be output in results? :P

**Comment by djmgit** [comment]
@vibhcool could you explain what you suggested about setting a lower limit?

**Comment by djmgit** [comment]
@kavithaenair this is strange, I am parsing the tweet text returned by loklak API, there should not be any HTML syntax in it, unless it is posted by any user. Could you please specify what you searched so that I can reproduce the problem :)

**Comment by vibhcool** [comment]
@djmgit , for some words in Stopwords list can be words of importance in Social Media. So in such situations they will be filtered out by the app. Could you set a lower limit like if that word's appearance is less than 10 (or something), then filter it, else show it?

**Comment by djmgit** [comment]
@vibhcool @singhpratyush @kavithaenair  I have filtered strings which consists of digits only and have also tried to remove symbols like # and @ in between words. Still there are some junk words. I will be dealing with them in separate subsequent issues as this PR deals with adding the word clicking feature.
Please re-review :)

**Pull request merged**